src/extractor/maneuver_override_relation_parser.cpp

```diff
@@ -103,18 +104,12 @@ ManeuverOverrideRelationParser::TryParse(const osmium::Relation &relation) const
         }
     }
 
-    if (from && (via || via_ways.size() > 0) && to)
+    if (from != SPECIAL_OSM_WAYID && to != SPECIAL_OSM_WAYID && via_node != SPECIAL_OSM_NODEID)
```

Comment:
This check won't work when there are only via ways right? In that case `via_node` will not be set only `via_ways`.

Reply:
As per chat, that is intended behavior, because a way is not sufficient to infer the instruction position.